spec/facts_spec.rb

```diff
@@ -67,7 +67,7 @@
     relative_path = Pathname.new(filepath).relative_path_from(project_dir).to_s
     describe relative_path do
       subject(:content) do
-        JSON.parse(File.open(filepath, 'rb') { |file| file.read })
+        JSON.parse(File.open(filepath, 'rb', &:read))
```

Comment:
Also here, is this better than `File.read()`?

Reply:
Converted to File.binread.